languages/c/tree-sitter/Parse_c_tree_sitter.ml

```diff
@@ -2419,3 +2419,10 @@ let parse file =
         { H.file; conv = H.line_col_to_pos file; extra = default_extra_env }
       in
       translation_unit env cst)
+
+let parse_pattern str =
+  let result = Parse_cpp_tree_sitter.parse_pattern str in
```

Comment:
maybe add a comment like (* Note that we're using Parse_cpp here! Not Parse_c, because XXXX *)

Reply:
done.